onboard-rest-api/src/main/java/no/vegvesen/ixn/serviceprovider/model/PrivateChannelApi.java

```diff
@@ -6,18 +6,23 @@ public class PrivateChannelApi {
     private String peerName;
     private String queueName;
 
+    private String serviceProviderName;
```

Comment:
Remove, it is not necessary in the request because we already have the serviceProviderName